src/app/features/issue/providers/azuredevops/azuredevops-issue/azuredevops-issue-map.util.ts

```diff
@@ -0,0 +1,44 @@
+import { AzuredevopsIssue, AzuredevopsIssueReduced } from './azuredevops-issue.model';
+import { AzuredevopsWorkItem } from '../azuredevops-api-responses';
+import { IssueProviderKey, SearchResultItem } from '../../../issue.model';
+
+export const mapAzuredevopsIssue = (issue: AzuredevopsWorkItem): AzuredevopsIssue => {
+  return {
+    id: issue.id,
+    url: issue.url,
+    number: issue.id,
+    state: issue.fields['System.State'],
+    title: issue.fields['System.Title'],
+    body: issue.fields['System.Description'],
+    created_at: issue.fields['System.CreatedDate'],
+    updated_at: issue.fields['System.ChangedDate'],
+    type: issue.fields['System.WorkItemType'],
+  };
+};
+
+export const mapAzuredevopsWIQLSearchResult = (res: any): AzuredevopsIssueReduced[] => {
+  return res.data.workItems.map(mapAzuredevopsReducedIssueFromWIQL);
+};
+
+export const mapAzuredevopsReducedIssueFromWIQL = (
+  workItem: any,
+): AzuredevopsIssueReduced => {
+  console.log(workItem);
```

Comment:
This was just there for debugging and should be removed in prod code then, should not it?